lib/pacemaker/pcmk_sched_primitive.c

```diff
@@ -544,6 +544,32 @@ create_migration_actions(pe_resource_t *rsc, const pe_node_t *current)
     }
 }
 
+/*!
+ * \internal
+ * \brief Abort a dangling migration by scheduling a stop (and possibly cleanup)
```

Comment:
Yep, see `unpack_migrate_to_success()`:
```
     * If the migrate_to and migrate_from both succeeded (which also implies the
     * resource is no longer running on the source), but there is no stop, the
     * migration is considered to be "dangling". Schedule a stop on the source
     * in this case.
     */
```

Reply:
I wonder if it would be worthwhile to have a glossary for such things in Pacemaker Developement

Reply:
Probably so. It's impossible to cover everything in a natural way in the exposition. It would be nice to have a glossary as an appendix.

If we do that, I'd prefer to go with an incremental approach rather than dedicating time to trying to build a comprehensive glossary.